src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/SetNoteStatusRequestHandlerTests.cs

```diff
@@ -80,12 +85,95 @@ public void ExternalUser_CanSetApprovedStatus(AuthorizationBuilder.UserType user
             A.CallTo(() => context.Notes.FindAsync(evidenceNoteId)).Returns(note);
 
```

Comment:
Line 50 should read "Non-external" as internal and unauth'd users are the ones who cannot access this area

also

Line73/74 Should this be a [Fact], as the test is only checking that the NoteStatus is approved and that the user is External